.github/workflows/gradle-dependency-submission.yml

```
@@ -2,7 +2,7 @@ name: Gradle Tests
 
 on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)